GameServerCore/Packets/Interfaces/IPacketNotifier.cs

```diff
@@ -35,8 +35,9 @@ public interface IPacketNotifier
         void NotifyDebugPacket(int userId, byte[] data);
         void NotifyEditBuff(IBuff b, int stacks);
         void NotifyEmotions(Emotions type, uint netId);
-        void NotifyEnterVision(IGameObject o, TeamId team);
-        void NotifyEnterVision(int userId, IChampion champion);
+        void NotifyEnterLocalVisibilityClient(IAttackableUnit unit, int userId = 0);
```

Comment:
I don't really like that `userId` is the last argument. I get why you did that (you need default value), but still it looks weird.

Reply:
Can I ignore this? I need the userId = 0 for the check if userId == 0, then broadcast. Perhaps we can discuss if EnterLocalVisibilityClient should Broadcast?

Reply:
add flag `IsBroadcast`.